zebra-scan/src/config.rs

```diff
@@ -0,0 +1,21 @@
+//! Configuration for blockchain scanning tasks.
+
+use indexmap::IndexMap;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
Optional: These traits are useful for testing.

```suggestion
#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
```